kratos/processes/find_intersected_geometrical_objects_process.h

```diff
@@ -7,355 +7,665 @@
 //  License:         BSD License
 //                     Kratos default license: kratos/license.txt
 //
-//  Main authors:    Pooyan dadvand
+//  Main authors:    Vicente Mataix Ferrandiz
```

Comment:
I don't think you are the main and unique author of this...

Reply:
This is probably for bad copy paste, changing. In any case this is in #4263

Reply:
Indeed, in the cpp the authorship is preserved at it is, this is result of some copy paste

~~~cpp
//  Main authors:    Pooyan Davand
//  Collaborators:   Ruben Zorrilla Martinez
//                   Vicente Mataix Ferrandiz
//
~~~